model/src/main/kotlin/DependencyNavigator.kt

```diff
@@ -161,7 +160,7 @@ interface DependencyNavigator {
         project: Project,
         maxDepth: Int = -1,
         matcher: DependencyMatcher = MATCH_ALL
-    ): SortedSet<Identifier> =
+    ): Set<Identifier> =
```

Comment:
Did you verify that all callers of this and the above function either sort on their own or don't require sorting?

Reply:
Yes, and I also consulted @oheger-bosch for this.